.flowconfig

```diff
@@ -11,6 +11,9 @@
 [include]
 <PROJECT_ROOT>/src
 
+[libs]
+<PROJECT_ROOT>/flow-typed/**/*.js
```

Comment:
It seemed like some files weren't getting picked up right before this change (namely argparse) so I added this to make sure they were.